src/classes/HiFiAudioAPIData.ts

```diff
@@ -399,7 +399,7 @@ export class HiFiAudioAPIData {
      * 
      * ✔ The client sends `hiFiGain` data to the server when `_transmitHiFiAudioAPIDataToServer()` is called.
      * 
-     * ✔ The server sends `hiFiGain` data to all clients connected to a server during "peer updates".
+     * x The server does not send `hiFiGain` data to all clients as part of "peer updates".
```

Comment:
@bridie-hifi Content looks good, but would you mind using the "* ❌" character that we use for the other ones? (e.g. attenuation below - you can just copy/paste)

Reply:
I think I fixed it!